account_financial_report/report/templates/journal_ledger.xml

```diff
@@ -306,10 +306,7 @@
                 </t>
             </div>
             <div class="act_as_cell amount" name="amount_currency">
-                <t
-                    t-if="move_line['amount_currency']"
-                    t-options="{'widget': 'float', 'precision': 2}"
-                >
+                <t t-if="move_line['amount_currency']">
```

Comment:
Why did you delete `t-option`?

Reply:
It was giving a qweb exception while printing the report. It didn't have any sense, because it's just used for formatting, but no output is ever made by a `<t t-if` node, so it was an obvious bug.